packages/ods-react/src/components/tag/src/dev.stories.tsx

```diff
@@ -0,0 +1,127 @@
+import { useState } from 'react';
+import { Tag } from '.';
+import { ICON_NAME, Icon } from '../../icon/src';
+import { Text } from '../../text/src';
+import { TAG_COLOR } from './constants/tag-color';
+import { TAG_SIZE } from './constants/tag-size';
+import { type TagRemoveEventDetail } from './components/tag/Tag';
+import { Button } from '../../button/src';
+export default {
+  component: Tag,
+  title: 'Tag dev',
+};
+
+export const Default = () => (
+  <Tag>
+    Tag
+  </Tag>
+);
+
+export const WithIcon = () => (
+  <Tag>
```

Comment:
try by applying class on tag directly (see Link to see how to set up classes on dev stories